site/docs-md/apis/accessibility/index.md

```diff
@@ -1,4 +1,4 @@
-<plugin-platforms platforms="ios,android"></plugin-platforms>
+<plugin-platforms platforms="pwa,ios,android"></plugin-platforms>
```

Comment:
@jcesarmobile - you mentioned to me the other day that "if it supports web/pwa, then it also supports electron". Is that the case? Hard to tell in some instances. For example, does Browser support electron even though there's no implementation file for it under electron folder?

Reply:
While it's true web support means basic electron support, for some APIs that doesn't qualify as true electron support (because what could be supported in electron is much more full featured than many web apis, for example)

Reply:
I think you can add electron support for local notifications, network, and app as recent additions

Reply:
done!

Reply:
Some of the pwa APIs are also limited compared with iOS/Android, but I think, as long as there is a working implementation we should mark it as supported, meaning, all the marked as pwa should also be marked as electron.